remoulade/api/main.py

```diff
@@ -47,6 +48,22 @@ def get_scheduled_jobs():
     return {"result": [job.as_dict() for job in scheduled_jobs.values()]}
 
 
+@app.route("/messages", methods=["POST"])
+def enqueue_message():
+    data = request.json
```

Comment:
could you use https://marshmallow.readthedocs.io/en/stable/ to validate the post data ?